pyproject.toml

```diff
@@ -83,6 +86,17 @@ all_writers = [
     "virtualizarr[icechunk]",
     "virtualizarr[kerchunk]",
 ]
+all = [
```

Comment:
These changes to non-docs dependency groups seem important and orthogonal to this PR?

Reply:
I removed these changes in [`70602f2` (#615)](https://github.com/zarr-developers/VirtualiZarr/pull/615/commits/70602f22814a160b30a3c85eaf14b5910086b739)